project/src/components/reviews-form/reviews-form.tsx

```diff
@@ -0,0 +1,38 @@
+import {useState} from 'react';
+import Rating from '../rating/rating';
+
+type FieldsType = {
+  target: {
+    name: string;
+    value: string;
+  }
+}
+
+function ReviewsForm(): JSX.Element {
+  const [formData, setFormData] = useState({
+    rating: '',
+    review: '',
+  });
+
+  const fieldChangeHandle = (evt: FieldsType) => {
```

Comment:
Д2 и в целом общепринятая практика 
```suggestion
  const handleFieldChange = (evt: ChangeEvent<......>) => {
```